templates/base.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html>
+  <head>
+    {% block head %}
+  <link rel="stylesheet" href="static/styles.css">
+  <title>{% block title %} Seats held by women {% endblock %}</title>
+    {% endblock %}
+  </head>
+  <body>
+
+<div class="grid-container">
+<header class="header"></header>
+<main class="main">{% block content %}{% endblock %}</main>
```

Comment:
is the `endblock` here necessary? There's nothing between `block` and `endblock`

Reply:
i think it's necessary bc that's the place where i can extends my base.html